common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1356 @@
+﻿################
+# air_group
+# earth_group
+# fire_group
+# water_group
+################
+blackrock_depths = {
+	county = c_blackrock_depths
+	
+	character_modifier = {
+		name = holy_site_blackrock_depths_effect_name
+		prowess = 2
+		heavy_infantry_toughness_mult = 0.3
+	}
+}
+balor = {
+	county = c_balor
+	
+	character_modifier = {
+		name = holy_site_balor_effect_name
+		prowess_per_piety_level = 1
+		supply_duration = 0.1
+	}
+}
+kubitis = {
+	county = c_kubitis
+	
+	character_modifier = {
+		name = holy_site_kubitis_effect_name
+		martial_per_piety_level = 1
+		raid_speed = 0.1
+	}
+}
+nordrassil = {
+	county = c_nordrassil
+	
+	character_modifier = {
+		name = holy_site_nordrassil_effect_name
+		monthly_piety_gain_mult = 0.1
+	}
+}
+maraudon = {
+	county = c_maraudon
+	
+	character_modifier = {
+		name = holy_site_maraudon_effect_name
+		knight_limit = 1
+	}
+}
+
+### theradras_worship
+lushwater = {
+	county = c_lushwater
+	
+	character_modifier = {
+		name = holy_site_lushwater_effect_name
+		prowess_per_piety_level = 1
+		clergy_opinion = 15
+	}
+}
+shehras = {
+	county = c_shehras
+	
+	character_modifier = {
+		name = holy_site_shehras_effect_name
+		levy_size = 0.15
+	}
+}
+#maraudon
+#eldrethalas
+thunder_bluff = {
+	county = c_thunder_bluff
+	
+	character_modifier = {
+		name = holy_site_thunder_bluff_effect_name
+		mountains_advantage = 3
+		hills_advantage = 3
+		stress_loss_mult = 0.2
+	}
+}
+
+### ordos_cult
+#xuen
+#niuzao
+osul = {
```

Comment:
no loc

![image](https://user-images.githubusercontent.com/46576860/99810284-8eebd700-2b54-11eb-92aa-990b6fc1dd6b.png)

Reply:
Changed in 41de90115c9c4582d8cd7c79aaaf57bf56a7871e.